apps/src/lib/levelbuilder/lesson-editor/AddResourceDialog.jsx

```diff
@@ -18,33 +60,212 @@ const styles = {
 export default class AddResourceDialog extends Component {
   static propTypes = {
     isOpen: PropTypes.bool.isRequired,
-    handleConfirm: PropTypes.func.isRequired
```

Comment:
I think you will want both handleClose and handleConfirm possibly because dialogs can be closed using the x without saving or closed using the save button.